compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -2064,4 +2064,11 @@ object messages {
     val explanation =
       hl"An object that contains ${"@static"} members must have a companion class."
   }
+
+  case class SymbolIsNotAValue(symbol: Symbol)(implicit ctx: Context) extends Message(SymbolIsNotAValueID) {
```

Comment:
`SymbolIsNotAValue ` -> `JavaSymbolIsNotAValue`
Same for ID.

Reply:
I changed the class name, ID and test name too.